devel/autoconf/Portfile

```diff
@@ -44,9 +44,16 @@ post-patch {
     touch ${worksrcpath}/man/autoreconf.1
 }
 
-# It should be safe to use the system Perl, since the scripts only use
-# core modules. The README prescribes 5.6 or later, and Tiger has 5.8.6.
-configure.perl      /usr/bin/perl
+if {${os.platform} eq "darwin"} {
+    # It should be safe to use the system Perl, since the scripts only use
+    # core modules. The README prescribes 5.6 or later, and Tiger has 5.8.6.
+    configure.perl  /usr/bin/perl
+} else {
+    depends_build-append \
+                    port:help2man
+    # https://trac.macports.org/ticket/70673
+    configure.perl  ${prefix}/bin/perl5
```

Comment:
help2man depends on perl5.34. perl5.34 provides the perl5.34 executable. The perl5 symlink is provided by the perl5 port. If you want to use the perl5 symlink here you have to depend on the perl5 port.